plugins/kotlin-as-java/src/main/kotlin/transformDocumentableToJava.kt

```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.dokka.kotlinAsJava
+
+import org.jetbrains.dokka.kotlinAsJava.converters.asJava
+import org.jetbrains.dokka.kotlinAsJava.transformers.JvmNameDocumentableTransformer
+import org.jetbrains.dokka.model.DClasslike
+import org.jetbrains.dokka.plugability.DokkaContext
+
+fun DClasslike.transformClasslikeToJava(context: DokkaContext): DClasslike {
+    return JvmNameDocumentableTransformer().transform(this.asJava(), context)
+}
```

Comment:
I have done it only for classlikes, since making a general function for all intermediate elements would be cumbersome (additional parameters for asJava functions), so they would have to be at least overloaded. WDYT @vmishenev?

Reply:
To my mind, this is ok.